src/ucp/am/eager_multi.c

```diff
@@ -167,7 +167,7 @@ ucp_am_eager_multi_zcopy_proto_init(const ucp_proto_init_params_t *init_params)
         .super.send_op       = UCT_EP_OP_AM_ZCOPY,
         .super.memtype_op    = UCT_EP_OP_LAST,
         .super.flags         = UCP_PROTO_COMMON_INIT_FLAG_SEND_ZCOPY |
-                               UCP_PROTO_COMMON_INIT_FLAG_ADJ_MAX_FRAG,
+                                      UCP_PROTO_COMMON_INIT_FLAG_CAP_SEG_SIZE,
```

Comment:
why need to add offset (here and in other places below)?

Reply:
in this case , need to ignore clang format

Reply:
fixed